quickwit-doc-mapping/src/wikipedia_mapper.rs

```diff
@@ -73,7 +73,9 @@ impl Default for WikipediaMapper {
 #[typetag::serde(name = "wikipedia")]
 impl DocMapper for WikipediaMapper {
     fn doc_from_json(&self, doc_json: &str) -> Result<Document, DocParsingError> {
-        self.schema.parse_document(doc_json)
+        self.schema
+            .parse_document(doc_json)
+            .map_err(DocParsingError::from)
```

Comment:
nitpick:
you don't need the map_err. That's what ? does.

```
self.schema.parse_document(doc_json)?;
Ok(())
```

Reply:
I need to check but I tried this way and visual studio was complaining

Reply:
I think you did not see that I added a custom DocParsingError, it's the same name of tantivy hence the confusion